src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -4,88 +4,103 @@ short-title: Integrate Flutter
+A Flutter module project will be created at `some/path/my_flutter/`. From that 
+directory, you can run the same `flutter` commands you would
+in any other Flutter project, like `flutter run --debug` or `flutter build ios`.
+You can also run the module in [Android Studio/IntelliJ][] or [VS Code][] with
+the Flutter and Dart plugins.
+This project contains a single-view example version of your module before it is
+embedded in your existing application, which is useful for incrementally
+testing the Flutter-only parts of your code.
 
-Assume you have an existing iOS app at `some/path/MyApp`, and that you
-want your Flutter project as a sibling:
+### Module organization
+The `my_flutter` module directory structure is similar to a normal Flutter
+application:
 
-```bash
-cd some/path/
-flutter create -t module my_flutter
+```text
+my_flutter/
+  .ios/
+    Runner.xcworkspace
+    Flutter/podhelper.rb
+  lib/
+    main.dart
+  test/
+  pubspec.yaml
 ```
 
-This creates a `some/path/my_flutter/` Flutter module project with some Dart
-code to get you started, and a `.ios/` hidden subfolder. The `.ios/` folder wraps
-the module project that contains some CocoaPods and a helper Ruby script.
+Add your Dart code to the `lib/` directory.
+
+Add Flutter dependencies to `my_flutter/pubspec.yaml`, including Flutter packages
+and plugins.
 
-## Make the host app depend on the Flutter module
+The `.ios/` hidden subfolder contains a Xcode workspace where you can 
+run a stand-alone version of your module, and is a wrapper project to bootstrap
+your Flutter code. It contains helper scripts to build frameworks or
+embed the module into your existing application with [CocoaPods][].
 
-The description below assumes that your existing iOS app has a structure similar
-to what you get by asking Xcode to generate a new "Single View App" project
-using Objective-C. If your existing app has a different folder structure
-and/or existing `.xcconfig` files, you can reuse those, but you'll probably need
-to adjust some of the relative paths mentioned below accordingly.
+{{site.alert.note}}
+Add custom iOS code to your existing application or a plugin, not to
+the module in `.ios/`. Changes made in `.ios/` are not embedded in your existing application.
+Regenerate the directory by running `flutter clean` or `flutter pub get` in the
+`my_flutter` directory.
+{{site.alert.end}}
 
-The assumed folder structure is as follows:
+## Embed the Flutter module in your existing application
+
+There are two ways to embed Flutter in your existing application.
+1. Use the CocoaPods dependency manager and installed Flutter SDK. Recommended.
+1. Create frameworks for the Flutter engine, your compiled Dart code, and all Flutter plugins.
+Manually embed the frameworks and update your existing application's build settings in Xcode.
+
+### Embed with CocoaPods and the Flutter SDK
+
+This method requires every developer working on your project to have a locally installed
```

Comment:
Add some pros and cons. e.g. you can just click run in xcode and not have to build 2 things when you're rapidly iterating.